run-tests.sh

```diff
@@ -15,44 +15,7 @@
 
 set -xeuo pipefail
 
-echo "Gate job started"
-echo "+-------------------- START ENV VARS --------------------+"
-env
-echo "+-------------------- START ENV VARS --------------------+"
+export RE_JOB_ACTION="${RE_JOB_ACTION}:-liberty_to_newton_leap"
```

Comment:
You'll want to wrap with the curly bracket:
`export RE_JOB_ACTION=${RE_JOB_ACTION:-liberty_to_newton_leap}`